transformer/catalog-transformer-xml/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD name="Xml Query Transformer"
+         description="Xml Response Queue Transformer"
+         id="ddf.catalog.transformer.xml.XmlResponseQueueTransformer">
+        <AD name="Parallel Marhsalling Threshold" id="threshold" required="true" type="Integer"
+            default="50"
```

Comment:
On my box, with the tests I ran, that seemed to be about nominal. It is very environment dependent.

Also, that's not the number of threads; that's the number of responses to a query above which we start splitting up the work. So a response of 202 metacards would be split into groups of

First 101 + 101
Then 51 + 50 + 51 + 50
Finally 26 + 25 + 50 + 26 + 25 + 50

(Something like that. I think the default for java division is CEIL.)

So there would be six total groups passed off to the FJP for processing where the FJP is being configured with the default of num-core threads.